onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -743,14 +765,27 @@ def test_output_match(self, device: str, dtype: torch.dtype, op):
                         # An onnxscript tensor
                         function_output = function_output.value
 
+                    actual = torch.tensor(function_output)
+                    expected = (
+                        torch_output
+                        if isinstance(torch_output, torch.Tensor)
+                        else torch.tensor(torch_output)
+                    )
+
+                    if op.name in NONDETERMINISTIC_OPS:
+                        # Check shape and dtype only for ops that are known to be
+                        # nondeterministic
+                        self.assertEqual(actual.shape, expected.shape)
```

Comment:
It'd better to verify the mean value between them are very close with an acceptable tolerance. [Here](https://github.com/pytorch/pytorch/blob/a641d6075761050599bf972c0b6bc3823182914e/test/onnx/test_pytorch_onnx_onnxruntime.py#L11792)is an example.

Reply:
Ops like empty are creating non initialized values. Or dropout which does not expect a matching mean. Should we distinguish them?

Reply:
I think I am inclined to create individual tests for distributions if desired

Reply:
Created https://github.com/microsoft/onnx-script/issues/467 to track